<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MatplotlibWidget" name="CCDImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RunButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="exposurebox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thresholdbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="MatplotlibWidget" name="WFimage">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>511</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Actuator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ActspinBox">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>530</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>70</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PushdoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>560</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="PushDMButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Push Actuator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveWFButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save WF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AutoScanButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ScanZernikeModeButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Zernike Mode</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ZernikeModeBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>70</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode#</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PushSpinBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>-10.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>560</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveZMWFButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save ZM WF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadDMButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load DM File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dmfilebox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="InitiatePushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Initiate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Infl_func_pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>600</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Infl_func</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
